docker/Dockerfile

```diff
@@ -17,3 +19,15 @@ RUN install2.r --error remotes &&  \
 
 COPY . /src
 RUN R CMD INSTALL --install-tests /src && rm -rf /src
+
+ENV ORDERLY_RUNNER_QUEUE_ID=orderly.runner
```

Comment:
a more informative name just to be safe ahha eg. `orderly.runner.queueId`

Reply:
gone with `orderly.runner.queue` for casing consistency